e2e/test-component/scenarios/embedding-sdk/interactive-question-theming.cy.spec.tsx

```diff
@@ -85,14 +85,6 @@ describe(
           assertBackgroundColorEqual($el, buttonHoverBg),
         );
 
-        // Hover should be a less lightened version of the background color.
-        cy.get(customColumn)
-          .should("be.visible")
-          .realHover()
-          .should(($el) =>
-            assertBackgroundColorEqual($el, lighten(BACKGROUND_COLOR, 0.4)),
-          );
```

Comment:
This is caused by `realHover` being super flaky, see this https://github.com/dmtrKovalenko/cypress-real-events/issues/691 as this is a Cypress problem. You can notice this easily by running the test in a loop locally, where the hover wouldn't be triggered correctly.

We also do not need this assertion anymore as we already have many others that checks that dynamic CSS variables work in this test.